docs/content/documentation/schema/concepts/_index.md

```diff
@@ -0,0 +1,98 @@
+---
+title: OSCAL Model Concepts
+heading: Concepts Within and Among OSCAL Models
+description: Reviews concepts within OSCAL models, as well as concepts across models.
+weight: 20
+sidenav:
+    title: OSCAL Model Concepts
+---
+
+Information is organized logically within each OSCAL model. OSCAL models are also deisnged to interact, which provides traceability and minimizes duplication.
+
+## General OSCAL Model Organization
+
+Every OSCAL model includes a required metadata section, optional back-matter section, and mode-specific body.
+
+## OSCAL File Organization
+
+Every OSCAL file must have a minimum set of required fields and assemblies, and must follow the [core OSCAL syntax](../).
+
+
+{{% usa-grid-container %}}
+{{% usa-grid-row %}}
+{{% usa-grid-column class="col-4" %}}
+&nbsp;
+
+![A diagram representing the general layout of any OSCAL file, showing the following information. A root element, a metadata section, a model-specific section, and a back-matter section.](OSCAL-file.svg)
+{{% /usa-grid-column %}}
+{{% usa-grid-column class="col-8, padding-left-2" %}}
+An OSCAL file is organized as follows:
+- **Root Element**: The root element of the file indicates the type of content within the body of the file. 
+
+- **UUID**: The Universally Unique ID (UUID) at the root must be changed every time the content of the file is modified.
+
+- **Metadata**: Every OSCAL file must have a metadata section and include a title, last-modified timestamp, and OSCAL syntax version. Metadata is also used to define roles, parties (people, teams and organizations), and locations.
+
+- **Model-specific Body**: The body of an OSCAL file is different for each model.
+
+- **Back Matter**: Back matter syntax is identical in all OSCAL models. It is used for attachments, citations, and embedded content such as graphics. Tool developers and content authors are encouraged to attach content here and reference it from within the body of an OSCAL file.
+
+
+
+{{% /usa-grid-column %}}
+{{% /usa-grid-row %}}
+
+{{% /usa-grid-container %}}
+
+## Catalog Model Concepts
+
+Many privacy and security compliance frameworks are based on **controls**. 
+Framework authors provide control requirements, system owners impliment the controls, and assessors validate the implemented control against the requirement. 
+
+Framework providers organize control requirements into a **catalog**.
```

Comment:
The Catalog heading seems to have been lost.

Reply:
I think that must have been a render glitch on your end. It is clearly present on line #47 and renders correctly on my workstation.